src/sasjsCli/loadDependenciesFile.ts

```diff
@@ -31,33 +33,38 @@ export const loadDependenciesFile = async ({
   macroFolders,
   buildSourceFolder,
   macroCorePath,
-  binaryFolders
-}: loadDependenciesParams) => {
+  binaryFolders,
+  compileTree
+}: LoadDependenciesParams) => {
```

Comment:
This must be a BREAKING CHANGE, as it is being used in [sasjs/server](https://github.com/sasjs/server/blob/main/api/scripts/compileSysInit.ts#L21).

Note: Is there a possibility to move `compileTree` logic entirely into `sasjs/utils`, like an internal thing?

Reply:
It can't be moved entirely to `@sasjs/utils`, because `CompileTree` has to be used across all files during the compilation.
Potentially it can be used optional. No, `CompileTree` is saved from cli https://github.com/sasjs/cli/blob/f1f6ffb2eed86d122c699b8af6459dd5da862c11/src/commands/compile/compile.ts#L241 .